packages/rocketchat-theme/server/server.js

```diff
@@ -0,0 +1,189 @@
+/* globals WebAppHashing */
+
+const less = Npm.require('less');
+const Autoprefixer = Npm.require('less-plugin-autoprefix');
+const crypto = Npm.require('crypto');
+const logger = new Logger('rocketchat:theme', {
+	methods: {
+		stop_rendering: {
+			type: 'info'
+		}
+	}
+});
+
+WebApp.rawConnectHandlers.use(function(req, res, next) {
+	let css;
+	let hash;
+
+	const path = req.url.split('?')[0];
+	const prefix = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX || '';
+	if (path === (`${ prefix }/__cordova/theme.css`) || path === (`${ prefix }/theme.css`)) {
+		css = RocketChat.theme.getCss();
+		hash = crypto.createHash('sha1').update(css).digest('hex');
+		res.setHeader('Content-Type', 'text/css; charset=UTF-8');
+		res.setHeader('ETag', `"${ hash }"`);
+		res.write(css);
+		return res.end();
+	} else {
+		return next();
+	}
+});
+
+const calculateClientHash = WebAppHashing.calculateClientHash;
+
+WebAppHashing.calculateClientHash = function(manifest, includeFilter, runtimeConfigOverride) {
+	let hash;
+	let themeManifestItem;
+	const css = RocketChat.theme.getCss();
+	if (css.trim() !== '') {
+		hash = crypto.createHash('sha1').update(css).digest('hex');
+		themeManifestItem = _.find(manifest, function(item) {
+			return item.path === 'app/theme.css';
+		});
+		if (themeManifestItem == null) {
+			themeManifestItem = {};
+			manifest.push(themeManifestItem);
+		}
+		themeManifestItem.path = 'app/theme.css';
+		themeManifestItem.type = 'css';
+		themeManifestItem.cacheable = true;
+		themeManifestItem.where = 'client';
+		themeManifestItem.url = `/theme.css?${ hash }`;
+		themeManifestItem.size = css.length;
+		themeManifestItem.hash = hash;
+	}
+	return calculateClientHash.call(this, manifest, includeFilter, runtimeConfigOverride);
+};
+
+RocketChat.theme = new class {
+	constructor() {
+		this.variables = {};
+		this.packageCallbacks = [];
+		this.files = ['server/lesshat.less', 'server/colors.less'];
+		this.customCSS = '';
+		RocketChat.settings.add('css', '');
+		RocketChat.settings.addGroup('Layout');
+		RocketChat.settings.onload('css', Meteor.bindEnvironment(() => {
+			return function(key, value, initialLoad) {
```

Comment:
There is no need for this `function`, pass the params directly to the above arrow function